public/app/features/panel/components/PanelPluginError.tsx

```diff
@@ -26,7 +26,7 @@ class PanelPluginError extends PureComponent<Props> {
 
     return (
       <div style={style}>
-        <Alert severity={AppNotificationSeverity.Error} {...this.props} />
+        <Alert severity={AppNotificationSeverity.Error} children={this.props.text} {...this.props} />
```

Comment:
This is temporary, right 😄

Reply:
I don't see why it would be? These error components are passed `text` which isn't used, because the props are incorrectly spread.

Reply:
Yes, I'm glad you fixed that - that we're not using `text` is something I noticed only after seeing your PR. I'm just talking about the `children` prop, which eslint is complaining about when you run the FE: 
>ERROR in [eslint] 
/Users/polibon/Documents/code/grafana/grafana/public/app/features/panel/components/PanelPluginError.tsx
  29:57  error  Do not pass children as props. Instead, nest children between the opening and closing tags  react/no-children-prop